src/System.Net.NetworkInformation/src/project.json

```diff
@@ -3,7 +3,8 @@
     "System.Diagnostics.Contracts": "4.0.0-*",
     "System.Threading": "4.0.0-*",
     "System.Net.Primitives": "4.0.10-*",
-    "System.Net.Sockets": "4.0.10-*"
+    "System.Net.Sockets": "4.0.10-*",
+    "System.Linq": "4.0.0"
```

Comment:
Oops, yeah, I somehow commented on the note above this one. It's used in a couple places at least.